monkey/agent_plugins/exploiters/hadoop/src/hadoop_exploiter.py

```diff
@@ -119,11 +119,23 @@ def _get_open_http_ports(target_host: TargetHost) -> Sequence[NetworkPort]:
     ]
 
 
+def _filter_out_closed_ports(
```

Comment:
Define methods/functions after they are used so you can easily read code from the top down.